### PR TITLE
Status transitions business stakeholders, users, user verification st…

### PR DESCRIPTION
**Pull request by arao6**
…atus transition

### CODE REVIEW

**Review by Rrathinasabapath — CHANGES_REQUESTED** [review]
Pls make changes per the comments.

File: src/test/java/com/hyperwallet/clientsdk/HyperwalletTest.java [hunk 1]
```diff
@@ -418,6 +418,236 @@ public void testGetUserStatusTransition_noTransitionToken() {
...
+            assertThat(e.getResponse(), is(nullValue()));
+            assertThat(e.getErrorMessage(), is(equalTo("User token may not be present")));
+            assertThat(e.getMessage(), is(equalTo("User token may not be present")));
+            assertThat(e.getHyperwalletErrors(), is(nullValue()));
+            assertThat(e.getRelatedResources(), is(nullValue()));
+        }
+    }
+
+    @Test
+    public void testUpdateUser_verificationStatus() throws Exception {
```
[the remark this answers is not in the document]
Comment (arao6) [reply src/test/java/com/hyperwallet/clientsdk/HyperwalletTest.java]: Renamed method

File: src/main/java/com/hyperwallet/clientsdk/Hyperwallet.java [hunk 2]
```diff
@@ -258,6 +256,54 @@ public HyperwalletBusinessStakeholder uploadDocumentBusinessStakeholder(String u
...
 
+    /**
+     * Create Business Stakeholder Status transition
+     *
+     * @param userToken   String
+     * @param stakeholderToken Hyperwallet Stakeholder token
+     * @param transition Hyperwallet Status Transition
+     * @return HyperwalletStatusTransition new status for Business Stakeholder
+     */
+    public HyperwalletStatusTransition createBusinessStakeholderStatusTransition(String userToken, String stakeholderToken, HyperwalletStatusTransition transition) {
```
[the remark this answers is not in the document]
Comment (arao6) [reply src/main/java/com/hyperwallet/clientsdk/Hyperwallet.java:267]: V4 has already been released, will be pushing post-release bug-fix and 1-2 changes, which have been merged into master.

**Review by Rrathinasabapath — APPROVED** [review]
Reviewed changes. Good to be reviewed by the next reviewer.

**Review by vkorotchenko — APPROVED** [review]

**Pull request merged**